### PR TITLE
Ensure strain is double precision

### PR DESCRIPTION
**Pull request by cdcapano**
This forces the strain retrieved by `from_cli` to be double precision if the precision argument is set to double. Also checks that the precision argument is either 'single' or 'double', raising an error otherwise.

### CODE REVIEW

File: pycbc/strain.py [hunk 1]
```diff
@@ -264,6 +264,11 @@ def from_cli(opt, dyn_range_fac=1, precision='single',
         if precision == 'single':
             logging.info("Converting to float32")
             strain = (strain * dyn_range_fac).astype(pycbc.types.float32)
+        elif precision == "double":
```
Comment (ahnitz) [inline pycbc/strain.py]: Should probably apply dyn_range_fac here too.

**Comment by cdcapano** [comment]
@ahnitz Currently the `DYN_RANGE_FACTOR` isn't multiplied to double precision strains though. If I change that, that may have an affect on other codes.

**Comment by ahnitz** [comment]
@cdcapano I don't think so, but it is currently wrong. In the one case I know of that uses double here, they don't provide a dynamic range factor, and by default it is 1. So the only case would be someone setting it and it is not currently working.

**Comment by cdcapano** [comment]
Ah, I didn't see that `dyn_range_fac` defaulted to 1. Ok, I added it. I'll merge as soon as Travis is done.

**Pull request merged**